services/src/test/scala/cromwell/pipeline/service/ProjectServiceTest.scala

```diff
@@ -35,6 +42,23 @@ class ProjectServiceTest extends AsyncWordSpec with Matchers with MockitoSugar w
       }
     }
 
+    "updateProjectName" should {
+      "return ok of a new project name" taggedAs Service in {
+        val newProjectName = dummyProject.copy(name = s"new${dummyProject.name}").name
```

Comment:
```suggestion
        val newProjectName = s"new${dummyProject.name}"
```
I think you over-engineered here a little bit

Reply:
Yes, you are right :)